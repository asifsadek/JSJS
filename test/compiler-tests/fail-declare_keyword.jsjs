val get = 5;
print(get);
