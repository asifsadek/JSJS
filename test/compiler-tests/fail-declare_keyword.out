Error: Cannot redefine keyword 'get'
